### PR TITLE
Adding servicegraph testing to postsubmit

### PR DESCRIPTION
**Pull request by utka**
Test will run for 10 min and query prometheus every minute, save the data in the file and copy it to gs:// bukcet for the daily build.

### CODE REVIEW

File: prow/perf_service_graph_test.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+CMD=""
+DELETE=""
+source "${ROOT}/prow/lib.sh"
+
+TIME_TO_RUN_PERF_TESTS=${TIME_TO_RUN_PERF_TESTS:-1200}
+
+pushd "${GOPATH}/src/istio.io/tools/perf/load"
+  WD="${GOPATH}/src/istio.io/tools/perf/load"
+  source common.sh
+  start_servicegraphs "1" "0"
```
Comment (howardjohn) [inline prow/perf_service_graph_test.sh]: Can you comment here what 1 and 0 are?

File: prow/perf_service_graph_test.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+perf_metrics="${OUTPUT_PATH}/perf_metrics.txt"
+rm "${perf_metrics}" || true
+
+pushd "${GOPATH}/src/istio.io/tools/perf/benchmark/runner"
+  count=$(expr $TIME_TO_RUN_PERF_TESTS / 60)
+  echo "Get metric $count time(s)."
+  for i in `seq 1 $count`;
+  do
+    sleep 1m
+    python prom.py http://localhost:8060 60 --no-aggregate >> ${perf_metrics}
```
Comment (howardjohn) [inline prow/perf_service_graph_test.sh]: So our end result will be 10 json objects?
```
{
... 1st minute
}
{
... 2nd minute
}
```

?

Just verifying.

Reading/analyzing these will be done in a later PR or do we have something for that now?

Comment (utka) [reply prow/perf_service_graph_test.sh]: Correct. 
We are not going to use the data here, just save it in a bucket together with all the artifacts for that particular daily build. 
I will set up daily cron job for reading the data in another script.

File: prow/perf_service_graph_test.sh (hunk 1 above)
Comment (utka) [reply prow/perf_service_graph_test.sh]: Done.

**Pull request merged**